keyboards/antari/antari.c

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2022 Sandipratama <https://github.com/nendezkombet>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "antari.h"
+
```

Comment:
This allows for a default behavior to be defined, without the need for the user to specify the behavior.  This is very useful for the configurator which doesn't have a way to configure this behavior, currently.

```suggestion

#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    }
    return true;
}
#endif
```

Reply:
@drashna awesome, how about Product and Vendor ID number ? can i just using random number for those and reuse the Vendor ID for my second project ?

Reply:
> how about Product and Vendor ID number ? can i just using random number for those and reuse the Vendor ID for my second project ?

That's been the general procedure for most vendor.  The further past 0x4000 for vendor IDs, the better.